Qabalany - De-News

qabalany opened the pull request:
<img width="322" height="92" alt="DENews logo" src="https://github.com/user-attachments/assets/321acf13-1e36-4c8c-bc34-a674ba72e87b" />



### **De-News – Decode the Future of Decentralization**

[https://denews.fartist.live](https://denews.fartist.live)

[denewswened.netlify.app](https://denewswened.netlify.app/)

news portal focused on Web3, blockchain, and decentralized technology. 


De-News – Decode the Future of Decentralization

### **This pull request includes the first version of the project, built with HTML and CSS only.**

**included:**

- Project folder structure

- Semantic HTML layout with header, main, and footer

- CSS styling for layout and design

- Responsive design for different screen sizes

- Basic accessibility (alt text, headings, contrast)

- JavaScript features will be added in later versions.

qabalany committed:
initial release with static version (no JavaScript features yet)

daniellauding commented on README.md:
Nice with the globe, and that i could spin. But it was a bit strange that my desktop cursor dissapeared on desktop :)

daniellauding commented on README.md:
When i tried the form it didnt work for me, i landed on https://httpbin.org/anything
<img width="1523" height="687" alt="image" src="https://github.com/user-attachments/assets/f39d4a31-369f-4b54-8a10-8d03edd053ca" />

daniellauding commented on index.html:
Recommended formatting:
- remove lines #70-71

daniellauding commented on index.html:
Why? Because i think they dont need to be there, easier to scan and read :D

daniellauding commented on index.html:
Also would use tab indent for those two lines

daniellauding commented on index.html:
Also would tab indent here :D

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Same

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on index.html:
Feels a bit off the formatting/indenting

daniellauding commented on style.css:
Great with some resetting! :)

daniellauding commented on style.css:
Variables is good! A nice way to structure design tokens

daniellauding commented on style.css:
Formatting a bit off from rest of css in the file, i would indent it more so it aligns with rest of the document

daniellauding commented on README.md:
Not sure you had the time – but didnt see any Javascript file to open / close a hamburger menu, i can see you now used pure CSS for it :)

daniellauding reviewed (approved):
Great stuff! Not bad :) Just some recommendations for future work

qabalany commented on README.md:
tried it again and it’s working. The link in the README doesn’t trigger it ,  you’ll need to fill out the form and click the “Send Message” button on the webpage 
<img width="1840" height="1007" alt="image" src="https://github.com/user-attachments/assets/738e57ac-cdc7-4ae8-bbac-10b62851c69c" />

qabalany commented on README.md:
Yes, that was one of the options. I didn’t realize it might feel a bit off, but I’ll keep it in mind when optimizing the 3D models next time.

<img width="394" height="430" alt="image" src="https://github.com/user-attachments/assets/f51adc6d-59dc-4cb4-827a-e2eba35dd95c" />

qabalany commented on README.md:
The box isn’t checked 🙂, I noted that further down.

<img width="628" height="271" alt="image" src="https://github.com/user-attachments/assets/74e47d6f-e502-480f-a244-444009574861" />

qabalany commented on index.html:
It looks correctly indented in VS Code on my side  maybe GitHub’s viewer is displaying it differently.

<img width="525" height="288" alt="image" src="https://github.com/user-attachments/assets/4e89a58e-992d-4489-a1f1-2d63ce8c64a2" />

qabalany commented:
@daniellauding  Wow, thanks for the incredibly detailed review! I really appreciate you going through everything so carefully, from the big stuff down to all the indentation. Those "Same" comments are gold for cleaning it all up.

Thanks again for all your help!

qabalany committed:
De-News 2.0, new Fartist.xyz domain, accessibility & JS improvements

qabalany committed:
Restored version with comprehensive enhancements

- Restored stable codebase to clean working state
- Enhanced CSS: Fixed formatting, improved browser compatibility
- Enhanced HTML: Added SEO optimization and accessibility features
- Enhanced JavaScript: Modular architecture with error handling
- Overall: Code cleanup, better maintainability, and improved UX

qabalany committed:
Final pre-submission updates: SVG icons, matrix animations, theme system, mobile nav improvements

qabalany committed:
Refine README and finalize project files

HIPPIEKICK reviewed (approved):
Hi Mohammad!
I was sure that I posted feedback to you a few weeks back, but now I see it's not there. So here goes ✨ 

This is quite advanced JavaScript for where we were in the course when you submitted this. You’ve explored some concepts we hadn’t fully covered yet, such as the this keyword, conditionals, objects, and try/catch blocks. Great that you want to explore new concepts!

With that said, it's important that you understand what’s going on in your code, especially if you’ve used AI. Do you feel confident explaining how each part works? If not, I recommend going through it step by step to make sure you fully grasp it.

Also - the code works well, but could it be made a bit easier to read or simplified in some places? Sometimes a bit of refactoring can help make complex code feel more approachable 😊

I'm approving this project, so my comments here are more of something to think about. I think you did a great job! Can you just make sure your Netlify link works so I can do a G/VG check on some visuals 😊

qabalany committed:
Update simplified JavaScript, HTML, and CSS files

qabalany commented:
Hi Matilda!

Thank you for the feedback! 😄 You’re absolutely right, the original solution you saw was more advanced than what we had covered at that point. I was eager to explore new concepts like this, objects, and try/catch, and spent my weekend diving in. I realized I had taken on more than I could handle at the time 😅.

To make it fully understandable for myself, I simplified the code. I remember removing the try/catch statements during that process, which actually hinted that something was off with my previous code push. Earlier, I had restored an older push version of the project, creating a conflict that required a force push for subsequent updates. I didn’t realize this at first and only noticed the README update, so my final JS never made it to the remote. I’ve now force-pushed the simplified version.

I really appreciate your guidance and suggestions about readability and refactoring, and I’ll keep them in mind as I continue practicing. My Netlify link has also been updated now.

Thank you again for your support! 🙏

HIPPIEKICK commented:
Thank you for sharing your reflection and process ⭐

> Hi Matilda!
> 
> Thank you for the feedback! 😄 You’re absolutely right, the original solution you saw was more advanced than what we had covered at that point. I was eager to explore new concepts like this, objects, and try/catch, and spent my weekend diving in. I realized I had taken on more than I could handle at the time 😅.
> 
> To make it fully understandable for myself, I simplified the code. I remember removing the try/catch statements during that process, which actually hinted that something was off with my previous code push. Earlier, I had restored an older push version of the project, creating a conflict that required a force push for subsequent updates. I didn’t realize this at first and only noticed the README update, so my final JS never made it to the remote. I’ve now force-pushed the simplified version.
> 
> I really appreciate your guidance and suggestions about readability and refactoring, and I’ll keep them in mind as I continue practicing. My Netlify link has also been updated now.
> 
> Thank you again for your support! 🙏
